lms/templates/courseware/xqa_interface.html

```diff
@@ -25,11 +25,11 @@
 
     $('#' + element_id + '_history_form').submit(
         function () {
-            var username = $("#" + element_id + "_history_student_username").val();
+            var username_or_email = $("#" + element_id + "_history_student_username").val();
```

Comment:
@asadiqbal08 Can you point me to the direction wherein HTML we are populating that field? 
is it a hidden field or it is exposed to the user? if is exposed to the user, can we change the id to `_history_student_identifier` to be precise?

thanks.

Reply:
also, can you change the label of the input field or add some placeholder? 
<img width="1035" alt="Screen Shot 2021-02-23 at 8 18 34 PM" src="https://user-images.githubusercontent.com/4252738/108864749-51479000-7614-11eb-846c-addc0268de3e.png">